libs/utils/WordPressUtils/src/test/java/org/wordpress/android/util/LogFileWriterTest.kt

```diff
@@ -0,0 +1,53 @@
+package org.wordpress.android.util
+
+import android.content.Context
+import android.os.Build
+import androidx.test.core.app.ApplicationProvider
+import java.io.FileReader
+import java.util.UUID
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.wordpress.android.util.helpers.logfile.LogFileProvider
+import org.wordpress.android.util.helpers.logfile.LogFileWriter
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [Build.VERSION_CODES.O_MR1])
+class LogFileWriterTest {
+    lateinit var testProvider: LogFileProvider
```

Comment:
Depending on what we end up doing with https://github.com/wordpress-mobile/WordPress-Android/pull/11536#discussion_r401942095 we should follow the same thing here. At the very least, it should be `private`.

Reply:
Changed this to `private` in a29f8bbe74ea811b0490a7be400bc6d0700deddf.

Same comment about mutability and this one has the added complexity that it's important to clean up the test directory after to avoid shared state.